.travis.yml

```diff
@@ -33,10 +34,13 @@ before_install:
       brew install yajl;
       brew install augeas --HEAD;
       brew install dbus;
-      brew install qt5;
+      # Building Qt5 currently takes too much time.
```

Comment:
> Please do not waste too much time on this, I made dbus optional: 4bf21f4

Great! I undid my changes to the Travis configuration and rebased this branch on `master`. After that I successfully built the Qt-GUI on my machine, using the bottled version of Qt 5. As far as I can tell this pull request is ready to be merged \o/. 

By the way: It looks like the build jobs did not trigger correctly after my rebase. Do  you know a method to manually start the Travis build job?